services/121-service/src/programs/fsp/fsp.service.ts

```diff
@@ -64,14 +64,20 @@ export class FspService {
 
     if (paymentResult.status === StatusEnum.succes) {
       for (let connection of details.connectionsForFsp) {
-        await this.storeTransaction(amount, connection, fsp, program, installment);
+        await this.storeTransaction(
+          amount,
+          connection,
+          fsp,
+          program,
+          installment,
+        );
       }
     }
 
     return { paymentResult, nrConnectionsFsp: details.paymentList.length };
   }
 
-  public async createPaymentDetails(
+  private async createPaymentDetails(
```

Comment:
is this expected? if so, then all good.

Reply:
Yes, i was showing/explaining Jannis the tests and we noticed that that method should be private.